lm_eval/tasks/tmmluplus/_generate_configs.py

```diff
@@ -0,0 +1,210 @@
+"""
+Take in a YAML, and output all "other" splits with this YAML
+"""
+import os
+import yaml
+import argparse
+
+from tqdm import tqdm
+
+# from lm_eval.logger import eval_logger
+
+import pandas as pd
+
+# Copy from https://github.com/iKala/ievals/blob/main/ievals/settings.py
+# from TMMLU+ offical example
+categories = {
+    "STEM": [
+        "physics",
+        "chemistry",
+        "biology",
+        "computer science",
+        "math",
+        "engineering",
+    ],
+    "humanities": ["history", "philosophy", "law"],
+    "social_sciences": [
+        "politics",
+        "culture",
+        "economics",
+        "geography",
+        "psychology",
+        "education",
+    ],
+    "other": ["other", "business", "health"], # (business, health, misc.)
```

Comment:
Is this `"other"` exhaustive yet? it seems like some of the subjects listed below are not yet in any grouping?

Reply:
Based on the original dataset, that was one of those issues I recall at [this comment].n the original dataset, that was the issues I just mention at [this comment](https://github.com/EleutherAI/lm-evaluation-harness/pull/1320#issuecomment-1901730239)